code/mission/missionlog.cpp

```diff
@@ -605,7 +605,7 @@ void message_log_add_seg(log_text_seg* entry, int x, int msg_color, const char*
 	entry->flags = flags;
 }
 
-void message_log_add_segs(const char* source_string, int msg_color, int flags = 0, SCP_vector<log_text_seg> *entry = nullptr)
+void message_log_add_segs(const char* source_string, int msg_color, int flags = 0, SCP_vector<log_text_seg> *entry = nullptr, bool split_string = false)
```

Comment:
Same question.

Reply:
It's used more than a handful of times and only one of those times (lua api) needs this parameter set to false. It makes more sense to leave it as optional, default true.

Reply:
Ok.